axelrod/strategies/worse_and_worse.py

```diff
@@ -0,0 +1,57 @@
+import random
+
+from axelrod import Actions, Player, random_choice
+
+C, D = Actions.C, Actions.D
+
+class WorseAndWorse (Player):
+    """
+    Defects with probability of 'current turn / 1000'. Therefore
+    it is more and more likely to defect as the round goes on.
+
+    Names:
+        - worse_and_worse: [PRISON1998]
+
+    """
+
+    name = 'Worse and worse'
+    classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': True,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def strategy(self, opponent):
+        current_round = len(self.history) + 1
+        if random.randint(0, 1000) < current_round:
```

Comment:
Can you rewrite this so that it's like the other strategy? IE so that is uses `random_choice`:

```
        probability = 1 - float(current_round) / 1000
        return random_choice(probability)
```

When you do that you can remove the `import random` at the top of this file.